core/src/test/scala/io/snappydata/app/CreateIndexTest.scala

```diff
@@ -16,33 +16,104 @@
  */
 package io.snappydata.app
```

Comment:
No DUnits? I think we need few Dunits to test both the insertion and query validation.